assets/sass/v2/_duo.scss

```diff
@@ -0,0 +1,7 @@
+.enroll-duo form,
+.mfa-verify-duo form {
```

Comment:
Add comment in another thread and would suggest to use V2 class name instead of V1.